### DISCUSSION TITLE
MountainDuck MacOS Mount Point

### DISCUSSION THREAD

**Discussion by clx69** [iterate-ch/cyberduck#16324] [Q&A] [answered]
Is it possible that the mount point for the Mountain Duck to be in a more "standard" location or in a directory in the user's home, just to be easier to access it also from terminal apps ?

**Comment by dkocher** [comment, marked as answer]
Refer to our [documentation](https://docs.mountainduck.io/mountainduck/preferences/#mount-location). Please restart Mountain Duck after changing the default.

**Reply by dkocher** [reply]
Restarting the application is no longer required with version 4.16.3 or later.
